frontend/src/Service/functions.js

```diff
@@ -12,3 +12,44 @@ export function AddAllCategory(statuses) {
     };
     return statuses;
 };
+
+export function convertIntToData(dateTime) {
+
+	let orderDate;
+
+	if (!dateTime) {
+		orderDate = new Date();
+	} else {
+		orderDate = new Date(dateTime * 1000);
+	};
+
+	const now = new Date();
+
+	orderDate = orderDate === null ? new Date() : orderDate;
+
+	const motht = (orderDate.getMonth() + 1) < 10 ? "0" + (orderDate.getMonth() + 1) : (orderDate.getMonth() + 1);
+	const date = orderDate.getDate() < 10 ? "0" + orderDate.getDate() : orderDate.getDate();
+	const hours = orderDate.getHours() < 10 ? "0" + orderDate.getHours() : orderDate.getHours();
+	const minutes = orderDate.getMinutes() < 10 ? "0" + orderDate.getMinutes() : orderDate.getMinutes();
+
+	let result;
+
+	if (now.getFullYear() === orderDate.getFullYear() && (now.getMonth() === orderDate.getMonth())) {
+		const dateDifference = orderDate.getDate() - now.getDate();
+		if (dateDifference === 0) {
+			result = "Today at";
+		} else if (dateDifference === 1) {
+			result = "Tomorrow at";
+		} else if (dateDifference === -1) {
+			result = "Yesterday at";
+		} else {
+			result = "" + date + "." + motht
+				+ "." + orderDate.getFullYear();
+		}
+	}
+
+	result += " " + hours + ":" + minutes;
+
+	return result;
+    
+};
```

Comment:
In the package.json, I noticed date-fns.
See what it can do:
```
import { format } from "date-fns";

// if you want to convert ISO date format (user.birth_date)
console.log(format("1957-09-12", 'dd.MM.yyyy'))

// if you want to convert a timestamp (restaurant.creation_date)
console.log(format(1551276256, 'dd.MM.yyyy HH:mm'))
```
output: 
```
12.09.1957
19.01.1970 01:54
```

You can use date-fns if you want. I think this is a more preferred option.

Examples:
```
import { format, formatDistance, formatRelative, subDays } from 'date-fns'

format(new Date(), "'Today is a' iiii")
//=> "Today is a Thursday"

formatDistance(subDays(new Date(), 3), new Date())
//=> "3 days ago"

formatRelative(subDays(new Date(), 3), new Date())
//=> "last Friday at 7:26 p.m."
```

[date-fns.org](https://date-fns.org/)